en/frames_vtol/vtol_tiltrotor_eflite_convergence_pixfalcon.md

```diff
@@ -24,19 +24,19 @@ Flight controller options ():
 
 ## Hardware Setup
 
-The vehicle needs 7 PWM signals, which must be connected to the flight controller outputs as specified in [Airframe Reference > VTOL Tiltrotor > E-flite Convergence](../airframes/airframe_reference.md#vtol_vtol_tiltrotor_e-flite_convergence).
-This mapping is reproduced below. 
+The vehicle needs 7 PWM signals, which must be connected to the flight controller outputs as specified in [Airframe Reference > VTOL Tiltrotor > E-flite Convergence](../airframes/airframe_reference.md#vtol_vtol_tiltrotor_e-flite_convergence). 
```

Comment:
@sfuhrer The eflight doesn't have a config any more. Would you just start from the second tailsitter option under https://docs.px4.io/main/en/airframes/airframe_reference.html#vtol-tiltrotor ?

This airframe doc doesn't say anything about the motors or support for elevons. I "imagine" you can map all the actuators as shown below, but can't try it. Can you confirm whether this might be possible using normal Actuator configuration steps. Any "gotchas"?

Basically I'm after enough information to know whether this can still be done, and to enable a beginner to do it, if they have the Actuator Config screen.

Reply:
Thanks. I'll update appropriately!

```suggestion
The vehicle needs 7 PWM signals for the motors and control surfaces:
- Motor (left/right/back)
- Tilt servos (right/left)
- Elevons (left/right)

These can be configured in the [Actuators Configuration](../config/actuators.md) by following the instructions for VTOL tiltrotor geometry and output configuration.
Note that you will need to start from the [Generic Tiltrotor VTOL](../airframes/airframe_reference.md#vtol_vtol_tiltrotor_generic_tiltrotor_vtol) frame.
```